docs/kagi/getting-started/setting-default.md

```diff
@@ -25,3 +25,6 @@ If you want to manually set Kagi as a default search engine, use these settings:
 - Kagi auto suggestions URL `https://kagi.com/api/autosuggest?q=%s`
 
 - Kagi auto suggestions URL (Privacy Pass) `https://kagisuggest.com/api/autosuggest?q=%s`
+
+- Reverse image search URL `https://kagi.com/reverse/upload`. 
```

Comment:
Does Vivaldi have the capability to set a "default" reverse image search URL? I'm unfamiliar with browsers having this functionality

Reply:
Yup, it has separate settings for default search engnine, default search engine for private windows, and default reverse image search (https://help.vivaldi.com/desktop/tools/search/#Search_images), which defaults to Bing.
But it allows you to set any search engine as long as you give it the URL
![image](https://github.com/user-attachments/assets/afb2ce40-8f8a-499d-a259-8dfd40f75ece)

Reply:
Huh alright. Do we know if other search engines support this? If not, may want to leave a note saying this is Vivaldi-specific